examples/analytics-iframe-transport-remote-frame.html

```diff
@@ -30,14 +30,18 @@
       };
 
       // Load the script specified in the iframe’s name attribute:
-      const url = JSON.parse(window.name).scriptSrc;
-      if (url && url.startsWith('https://3p.ampproject.net/')) {
-        script = document.createElement('script');
-        script.src = url;
-        document.head.appendChild(script);
-        // The script will be loaded, and will call onNewContextInstance()
-      } else {
-        console.warn('Received invalid URL - risk of XSS! ' + url);
+      try {
+        var url = JSON.parse(window.name).scriptSrc;
+        if (url && url.startsWith('https://3p.ampproject.net/')) {
+          script = document.createElement('script');
+          script.src = url;
+          document.head.appendChild(script);
+          // The script will be loaded, and will call onNewContextInstance()
+        } else {
+          console.warn('Received invalid URL - risk of XSS! ' + url);
+        }
+      } catch (e) {
+        console.warn('Caught exception parsing window.name!');
```

Comment:
Make this:  try { console.warn('Caught...', e); } catch (err) {}

Reply:
Done. (I assume you are just saying to log the value of the error, not to move the warning from the catch into the try...)